hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/store/StateStoreService.java

```diff
@@ -64,6 +64,12 @@
  * <li>File {@link
  * org.apache.hadoop.hdfs.server.federation.store.driver.impl.StateStoreFileImpl
  * StateStoreFileImpl}
+ * <li>FileSystem {@link
+ * org.apache.hadoop.hdfs.server.federation.store.driver.impl.StateStoreFileSystemImpl
+ * StateStoreFileSystemImpl}
```

Comment:
Might be good to further distinguish these two. The former using for the local file system and the other using a generic filesystem.

Reply:
I think it might be better to do it like you do, but from a code perspective, these names correspond better to Java classes.

Reply:
True. I think we can leave it as it is in your PR.